internal/kafka/internal/services/kafka_tls_certificate_management/kafka_tls_certificate_management_config.go

```diff
@@ -0,0 +1,131 @@
+package kafka_tls_certificate_management
+
+import (
+	"fmt"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/environments"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+	"github.com/caddyserver/certmagic"
+	"github.com/go-playground/validator/v10"
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+)
+
+const (
+	inMemoryStorageType            = "in-memory"
+	secureStorageType              = "secure-storage"
+	fileStorageType                = "file"
+	manualCertificateManagement    = "manual"
+	automaticCertificateManagement = "automatic"
+)
+
+var validStorageTypes = []string{inMemoryStorageType, fileStorageType, secureStorageType}
+var validCertificateManagementStrategies = []string{manualCertificateManagement, automaticCertificateManagement}
+
+type KafkaTLSCertificateManagementConfig struct {
+	CertificateAuthorityEndpoint         string
+	CertificateManagementStrategy        string
+	StorageType                          string
+	EnableKafkaExternalCertificate       bool
+	ManualCertificateManagementConfig    ManualCertificateManagementConfig
+	AutomaticCertificateManagementConfig AutomaticCertificateManagementConfig
+}
+
+type ManualCertificateManagementConfig struct {
+	KafkaTLSCert     string
+	KafkaTLSCertFile string `validate:"required"`
```

Comment:
I find the "file" suffix confusing. Is it the content of the file? or is it a path?
Could we rename them to contain the "FilePath" suffix to make clear what it is?